### PR TITLE
Update to last version of slingshot

### PR DESCRIPTION
**Pull request by iLeonelPerea**
closes #142 

changes proposed in the pull request:

- Update dependences
- Fix typos
- Add babel plugins for production

@thetonymaster

### CODE REVIEW

**Comment by iLeonelPerea** [comment]
I removed codeclimate webhook..

File: appveyor.yml [hunk 1]
```diff
@@ -26,9 +27,9 @@ build: off
 notifications:
 - provider: Email
   to:
-  - leonel@tinkerware.io
-  subject: 'Build failed: provisioner-website'
-  message: The continuous integration build failed. See https://ci.appveyor.com/project/iLeonelPerea/provisioning-webpage/ for details.
+  - housecor@gmail.com
```
Comment (thetonymaster) [inline appveyor.yml]: change this to your mail

File: appveyor.yml [hunk 2]
```diff
@@ -26,9 +27,9 @@ build: off
...
+  subject: 'Build failed: react-slingshot'
+  message: The continuous integration build failed. See https://ci.appveyor.com/project/CoryHouse/react-slingshot/ for details.
```
Comment (thetonymaster) [inline appveyor.yml]: And this URL too.

**Comment by thetonymaster** [comment]
LGTM

Just fix the comments.

**Pull request merged**